packages/cli/src/models/files/ZosNetworkFile.ts

```diff
@@ -77,6 +77,11 @@ export default class ZosNetworkFile {
     return file ? file.zosversion : null;
   }
 
+  public static getDependencies(fileName: string): { [key: string]: any } | undefined {
```

Comment:
Can we remove this method, and use something like `packageFile.networkFile(network).dependencies`? I'd assume that networkFile for a new network would yield an empty ZosNetworkFile, that would have empty dependencies as well.